packages/browser/src/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.ts

```diff
@@ -59,36 +61,57 @@ export default class AuthorizationCodeWithPkceOidcHandler
   }
 
   async handle(oidcLoginOptions: IOidcOptions): Promise<void> {
-    const requestUrl = new URL(
-      oidcLoginOptions.issuerConfiguration.authorizationEndpoint.toString()
-    );
-    const codeVerifier = await this.joseUtility.generateCodeVerifier();
-    // Disable camel case rule because this query requires camel case
+    const redirector = this.redirector;
+    const storage = this.storageUtility;
+
     /* eslint-disable @typescript-eslint/camelcase */
-    const query: { [key: string]: string } = {
-      response_type: "code id_token",
-      redirect_uri: oidcLoginOptions.redirectUrl.toString(),
-      // TODO: the 'webid' scope does not appear in the specification
-      // A question regarding its use has been filed https://github.com/solid/specification
```

Comment:
It's a shame that it doesn't link to the actual question, but this comment still seems relevant?